src/client/webpack.config.js

```diff
@@ -16,7 +16,8 @@ const plugins = [
   new WebpackMessages({ name: pkg.name }),
   new webpack.NoEmitOnErrorsPlugin(),
   new webpack.DefinePlugin({
-    "process.env.NODE_ENV": JSON.stringify(ENV)
+    "process.env.NODE_ENV": JSON.stringify(ENV),
```

Comment:
by convention, `NODE_ENV` is a string with the name of the environment, like `development`, `production`, etc.